src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+import model.Ball;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int randomNumber = new Random().nextInt(100);

```

Comment:
Creating a new instance of Random for every method call is inefficient. Consider creating a single instance of Random as a private class field and reuse it.